venues/DBLP.org/python/superuser-init.py

```diff
@@ -112,7 +112,7 @@ def overwrite_allowed(groupid):
                 'description': 'Comma separated list of author email addresses, in the same order as above.',
                 'order': 3,
                 'values-regex': "[^;,\\n]+(,[^,\\n]+)*",
-                'required':True
+                'required':False
```

Comment:
When we add a revision to an existing record, if this is required the ["UNK"] values are inserted which show up for the paper rather than the existing (good) data already in the database.